pkg/controller/mongodb/mongodb_controller.go

```diff
@@ -57,6 +57,8 @@ const (
 	clusterFilePath                = "/var/lib/automation/config/automation-config"
 	operatorServiceAccountName     = "mongodb-kubernetes-operator"
 	agentHealthStatusFilePathValue = "/var/log/mongodb-mms-automation/healthstatus/agent-health-status.json"
+
+	hasLeftReadyState = "mongodb.com/v1.hasLeftReadyState"
```

Comment:
We store the other annotation keys in `mongodb_types.go` and should stay consistent about this and their naming, e.g. `hasLeftReadyStateAnnotationKey`. In my eyes they do make more sense in the controller file though so perhaps we should move `LastVersionAnnotationKey` over!

Reply:
I was thinking the same thing, I can move them over here.